spark2/src/main/java/org/apache/iceberg/spark/source/Reader.java

```diff
@@ -520,10 +500,9 @@ private InternalRowReaderFactory() {
     }
 
     @Override
-    public InputPartitionReader<InternalRow> create(CombinedScanTask task, Schema tableSchema, Schema expectedSchema,
-                                                    String nameMapping, FileIO io,
-                                                    EncryptionManager encryptionManager, boolean caseSensitive) {
-      return new RowReader(task, tableSchema, expectedSchema, nameMapping, io, encryptionManager, caseSensitive);
+    public InputPartitionReader<InternalRow> create(CombinedScanTask task, Table table,
```

Comment:
Spacing on this and the method below seems odd ... I think expectedSchema would fit on the previous line? I don't really mind though but usually this sort of thing bugs you :) Fine as is if you don't want to just have a single dangling caseSensitive

Reply:
I am paranoid about such things :) It feels a bit unbalanced with only caseSensitive on a new line.

<img width="867" alt="image" src="https://user-images.githubusercontent.com/6235869/115338554-e4ee9600-a157-11eb-8815-38e00842c83c.png">

<img width="772" alt="image" src="https://user-images.githubusercontent.com/6235869/115338596-fafc5680-a157-11eb-94b5-c78180a4d6ad.png">